Sources/Swift/Helper/SentryMobileProvisionParser.swift

```diff
@@ -0,0 +1,63 @@
+@objc @_spi(Private)
+public class SentryMobileProvisionParser: NSObject {
+    private var provisionsAllDevices: Bool = false
+    private var embeddedProfilePath: String?
+    
+    // If the profile provisions all devices, it indicates Enterprise distribution
+    @objc 
+    public var mobileProvisionProfileProvisionsAllDevices: Bool {
+        return provisionsAllDevices
+    }
+    
+    @objc
+    convenience override public init() {
+        self.init(nil)
+    }
+    
+    @objc
+    public init(_ path: String?) {
```

Comment:
`l`: Maybe you can add a comment why we make this nullable for testability and not having to pass in the path from the crash monitor system